src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const rows = table.rows;
+const cellFromIndex = 1;
+const cellToIndex = rows[0].cells.length - 1;
```

Comment:
The `cellToIndex` is calculated as `rows[0].cells.length - 1`, which means the cloned cell will be inserted before the last cell. If the intention is to insert the cloned cell at the end of the row, consider using `rows[0].cells.length` instead.